pkg/cosign/bundle/tsa.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+// TSABundle holds metadata about timestamp RFC3161 verification data.
+type TSABundle struct {
+	// SignedRFC3161Entry contains a RFC3161 signed timestamps provided by a time-stamping server.
+	// Clients MUST verify the hashed message in the message imprint
+	// against the signature in the bundle. This is encoded as base64.
+	SignedRFC3161Entry []byte
+
+	// TODO: Additional fields fir verification data such as CertBytes
+}
+
+func EntryToTSABundle(timestampRFC3161Entry []byte) *TSABundle {
```

Comment:
```suggestion
func TimestampToTSABundle(timestampRFC3161 []byte) *TSABundle {
```

Reply:
I don't think we should use the term 'entry' given the timestamp isn't admitted into a log

Reply:
Whenever it gets stored in a tlog, it'll make more sense. Also we could suffix it with response as it contains information about the raw taken, hashed message, time...